registry/coder/modules/digitalocean-region/README.md

```diff
@@ -0,0 +1,155 @@
+  version = "1.0.0"
+}
+
+resource "digitalocean_volume" "home_volume" {
+  region = module.region.value
+  # ... other configuration
+}
+
+resource "digitalocean_droplet" "workspace" {
+  region = module.region.value
+  # ... other configuration
+}
+```
+
+### Customized Configuration
+
+```tf
+module "region" {
+  source  = "registry.coder.com/coder/digitalocean-region/coder"
+  version = "1.0.0"
+
+  default      = "sfo3"
+  mutable      = true
+  display_name = "Datacenter Location"
+  description  = "Select the datacenter region for your development environment"
+}
+
+resource "digitalocean_droplet" "workspace" {
+  region = module.region.value
+  # ... other configuration
+}
+```
+
+## Available Regions
+
+The module includes the following DigitalOcean regions:
+
+| Region Code | Location                       | Notes        |
+| ----------- | ------------------------------ | ------------ |
+| `tor1`      | Canada (Toronto)               | 🇨🇦           |
+| `fra1`      | Germany (Frankfurt)            | 🇩🇪           |
+| `blr1`      | India (Bangalore)              | 🇮🇳           |
+| `ams3`      | Netherlands (Amsterdam)        | 🇳🇱 (Default) |
+| `sgp1`      | Singapore                      | 🇸🇬           |
+| `lon1`      | United Kingdom (London)        | 🇬🇧           |
+| `sfo2`      | United States (California - 2) | 🇺🇸           |
+| `sfo3`      | United States (California - 3) | 🇺🇸           |
+| `nyc1`      | United States (New York - 1)   | 🇺🇸           |
+| `nyc3`      | United States (New York - 3)   | 🇺🇸           |
+
+> **Note**: Some regions (nyc1, sfo1, ams2) are excluded because they do not support volumes, which are commonly used for persistent data storage.
+
+## Variables
+
+| Variable       | Type     | Default                                                      | Description                                                |
+| -------------- | -------- | ------------------------------------------------------------ | ---------------------------------------------------------- |
+| `default`      | `string` | `"ams3"`                                                     | The default region to select                               |
+| `mutable`      | `bool`   | `false`                                                      | Whether the region can be changed after workspace creation |
+| `name`         | `string` | `"region"`                                                   | The name of the parameter                                  |
+| `display_name` | `string` | `"Region"`                                                   | The display name of the parameter                          |
+| `description`  | `string` | `"This is the region where your workspace will be created."` | The description of the parameter                           |
+| `icon`         | `string` | `"/emojis/1f30e.png"`                                        | The icon to display for the parameter                      |
```

Comment:
Remove this section as we already parse and render Variables on the registry frontend.